help.txt

```diff
@@ -13,13 +13,11 @@ asdf plugin update --all                Update all plugins
 
 
 MANAGE PACKAGES
+asdf install                            Install all the package versions listed
+                                        in the .tool-versions file 
+asdf install [<name>]                   Install one tool at the version
+                                        specified in the .tool-versions file
 asdf install [<name> <version>]         Install a specific version of a package
```

Comment:
If this is a legal usage

```shell
asdf install gcloud 299.0.0 firebase 8.5.0
```

Then should 

```shell
asdf install gcloud firebase
```

not also be legal when given a `.tool-versions` of

```
gcloud 299.0.0
firebase 8.5.0
```

?

Reply:
Ignore this thread. See other comment